### DISCUSSION TITLE
using ExWebRTC in Docker?

### DISCUSSION THREAD

**Discussion by rgfaber** [elixir-webrtc/ex_webrtc#175] [Q&A]
Hi everyone,

I am trying to run an Elixir WebRTC Peer from a docker container, but I'm stuck at the point of NIF execution.

I've created a minimal sample application [here](https://github.com/beam-campus/ex_webrtc_docker_debug) 

I'm very new to this matter, so if anyone could point me in the right direction, much appreciated!

**Comment by mickel8** [comment]
Hi @rgfaber, I haven't looked at your Dockerfiles yet, but did you see ours :D ? For example: https://github.com/elixir-webrtc/apps/blob/master/nexus/Dockerfile

**Comment by mickel8** [comment]
You might be missing some native dependencies. Please take a look at our Dockerfile and see whether you have srtp and openssl installed :)
